api/controllers/transactions.js

```diff
@@ -74,7 +74,7 @@ TransactionsController.getTransactions = async function(context, next) {
 		senderId: params.senderId.value,
 		senderPublicKey: params.senderPublicKey.value,
 		type: params.type.value,
-		height: params.height.value,
+		blockHeight: params.height.value,
```

Comment:
what if `blockHeight` gets changed to something else tomorrow? how do we know it works that time?
how do we ensure these filters are intact?

Reply:
@diego-G There is a test for this but it's returning false positive

https://github.com/LiskHQ/lisk/blob/1.5.0/test/functional/http/get/transactions.js#L575

Reply:
This has been addressed with help of @lsilvs